docs/man-openmpi/man1/mpirun.1.rst

```diff
@@ -254,7 +254,7 @@ processes will be bound to the socket.
   program on the given nodes.  This option indicates that the
```

Comment:
Can you re-arrange the options in line 253 to show `-n` first?  And then add an `.. info::` block to say that `-n` is preferred, but the other forms are also accepted?

Reply:
THX, I just pushed an update for this.